commands/apps_dev.go

```diff
@@ -274,19 +288,19 @@ func RunAppsDevBuild(c *CmdConfig) error {
 		return err
 	}
 
-	if Interactive {
-		choice, err := confirm.New(
-			"start build?",
-			confirm.WithDefaultChoice(confirm.Yes),
-			confirm.WithPersistPrompt(confirm.PersistPromptIfNo),
-		).Prompt()
-		if err != nil {
-			return err
-		}
-		if choice != confirm.Yes {
-			return fmt.Errorf("cancelled")
-		}
-	}
+	// if Interactive {
```

Comment:
uncomment?

Reply:
commented out temporarily until we add more useful output prior or "do we need to ask for confirmation" logic